config/variables/factions.yml

```diff
@@ -1,18 +1,18 @@
-1:
+"1":
```

Comment:
Why exactly is that required?

Reply:
Because the `HasLookupAttributes` uses strings to do the lookup -- this was the one exception where actual row IDs were used (which is super-fragile) 

By forcing them to be strings -- the lookup works for Faction as well...

Reply:
Okay. Good to know :)